src/System.Diagnostics.FileVersionInfo/pkg/System.Diagnostics.FileVersionInfo.pkgproj

```diff
@@ -3,13 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Diagnostics.FileVersionInfo.csproj">
-      <SupportedFramework>net46;netstandardapp1.5</SupportedFramework>
+      <SupportedFramework>net46;netcore50;netstandardapp1.5</SupportedFramework>
```

Comment:
Previously we didn't have a UWP impl due to native APIs missing from api-sets/WACK-compliant surface.  Are we now using the Metadata-reader based implementation on UWP?